cloud/docker/_docker.py

```diff
@@ -562,6 +562,8 @@ def _human_to_bytes(number):
 
     if isinstance(number, int):
         return number
+    if number.isdigit():
+        return int(number)
```

Comment:
ah, sorry, you are correct, i had not read enough of the context

outside scope of this PR: note to move this to use the common human/byte functions